file_mapper_script.py

```diff
@@ -176,18 +206,36 @@ def parse_data(contents, verbose=False, testdebug=False):
                 elif args.overwrite:
                     do_action(source, destination, args.action,
                     overwrite=args.overwrite, testdebug=args.testdebug, relsym=args.relative_symlink)
+                    if args.sidecars:
+                        json_src, json_dest = map_sidecars(source, destination)
+                        do_action(json_src, json_dest, args.action,
+                                  testdebug=args.testdebug,
+                                  relsym=args.relative_symlink)
                     if verbose:
                         print("File has been overwritten")
                 elif os.path.exists(dirname):
                     if verbose:
                         print("Path already exists: " + str(dirname))
             elif os.path.isdir(os.path.dirname(destination)):
                 do_action(source, destination, args.action, testdebug=args.testdebug, relsym=args.relative_symlink)
+                if args.sidecars:
+                    json_src, json_dest = map_sidecars(source, destination)
+                    do_action(json_src, json_dest, args.action,
+                              testdebug=args.testdebug,
+                              relsym=args.relative_symlink)
             else:
                 os.makedirs( dirname )
                 if verbose:
                     print("Path has been made: " + str(dirname))
                 do_action(source, destination, args.action, testdebug=args.testdebug, relsym=args.relative_symlink)
+                if args.sidecars:
+                    json_src, json_dest = map_sidecars(source, destination)
+                    do_action(json_src, json_dest, args.action,
+                              testdebug=args.testdebug,
+                              relsym=args.relative_symlink)
+        elif "run-" not in source:
+            print("Could not create map. Check source path")
+            print(f"Source path: {source}")
```

Comment:
Not every `source` should have `run-` in it.  What is the intent for this `elif` block?  Could you maybe add a one-line comment above the `elif` to describe the intent?

Reply:
Hi @ericearl My apologies...This is code from some other tests and has been removed. It should not be included. In fact the json sidecar code was meant to be a separate pull request after making initial contact to see if your group was open to input. Sorry for the messy pull request. It won't happen again.